core/chains/evm/gas/fixed_price_estimator.go

```diff
@@ -89,6 +107,28 @@ func (f *fixedPriceEstimator) GetDynamicFee(_ context.Context, originalGasLimit
 	}, chainSpecificGasLimit, nil
 }
 
-func (f *fixedPriceEstimator) BumpDynamicFee(_ context.Context, originalFee DynamicFee, originalGasLimit uint32, maxGasPriceWei *assets.Wei, _ []EvmPriorAttempt) (bumped DynamicFee, chainSpecificGasLimit uint32, err error) {
-	return BumpDynamicFeeOnly(f.config, f.bhConfig.EIP1559FeeCapBufferBlocks(), f.lggr, f.config.TipCapDefault(), nil, originalFee, originalGasLimit, maxGasPriceWei)
+func (f *fixedPriceEstimator) BumpDynamicFee(
+	_ context.Context,
```

Comment:
Why do we pass around unused context to these methods? Should we consider just dropping it?

Reply:
i think this is conforming to the standard EVM estimator interface. other estimators like block history or L2 make calls to external services if i remember correctly